templates/store/publisher-details.html

```diff
@@ -190,9 +190,11 @@ <h4 class="p-media-object--snap__title">
   {# templates #}
   {% set article_title = {'rendered': '${title}'} %}
   {% set article = {'slug': '${slug}', 'title': article_title} %}
-  <script type="text/template" id="blog-article-template">
-    {% include 'partials/blog-card--minimal.html' %}
-  </script>
+  {% if blog_slug %}
```

Comment:
Or you could wrap the script tags I said to wrap? These can be wrapped too, but won't stop the errors.